BaseFilter/sections/css_extended.txt

```diff
@@ -2424,8 +2424,8 @@ yeptube.com#?#.wrapper > div.container:has(> div.c-title > div.c-medtitle-output
 ! facebook.com
 ! https://github.com/AdguardTeam/AdguardFilters/issues/135446#issuecomment-1332051619
 ! Facebook Marketplace
-[$path=/marketplace/]facebook.com#$?#div[class][style^="max-width:"]:has(> span > div  a[href^="/ads/about/"]) { display: none !important; }
-[$path=/marketplace/]facebook.com#$?#div[class][style^="max-width:"]:has(> span > div > a[target="_blank"]:not([href^="/marketplace/"])) { display: none !important; }
+[$path=/marketplace/]facebook.com,[$path=/marketplace/]facebookwkhpilnemxj7asaniu7vnjjbiltxjqhye3mhbshg7kx5tfyd.onion#$?#div[class][style^="max-width:"]:has(> span > div  a[href^="/ads/about/"]) { display: none !important; }
```

Comment:
`$path` don't work after duplicate :( - first is used on all listed domains.

Reply:
At that point, I think it's better to wait to hear from AdGuard devs about what `$path` is and how it works like.

Reply:
Incorrect rule.
Correct - with one modifier:

```adblock
[$path=/marketplace/]facebook.com,facebookwkhpilnemxj7asaniu7vnjjbiltxjqhye3mhbshg7kx5tfyd.onion#$?#div[class][style^="max-width:"]:has(> span > div  a[href^="/ads/about/"]) { display: none !important; }
```

![image](https://user-images.githubusercontent.com/8361299/222120711-e5e71c1f-730a-4db9-883b-a25a19b55e6d.png)